cmd/tk/flags.go

```diff
@@ -58,7 +58,7 @@ func labelSelectorFlag(fs *pflag.FlagSet) func() labels.Selector {
 func jsonnetFlags(fs *pflag.FlagSet) func() tanka.JsonnetOpts {
 	getExtCode, getTLACode := cliCodeParser(fs)
 	maxStack := fs.Int("max-stack", 0, "Jsonnet VM max stack. The default value is the value set in the go-jsonnet library. Increase this if you get: max stack frames exceeded")
-	jsonnetImplementation := fs.String("jsonnet-implementation", "go", "Only go is supported for now.")
+	jsonnetImplementation := fs.String("jsonnet-implementation", "go", "Use `go` to use native go-jsonnet implementation and `binary:<path>` to delegate evaluation to a binary (with the same API as the regular `jsonnet` binary)")
```

Comment:
> (with the same API as the regular `jsonnet` binary)

Could you be a bit more specific about what's required from these binaries? Which commands required and with what flags? Asking for the CLI to be exactly the same is probably a bit much, right?